#goal definition
string turn_direction
---
#feedback definition
string feedback
---
#result definition
string result
